crates/ra_syntax/src/ast/generated.rs

```diff
@@ -108,6 +108,10 @@ impl ArrayExpr {
     pub fn exprs(&self) -> impl Iterator<Item = &Expr> {
         super::children(self)
     }
+
+    pub fn repeat(&self) -> Option<&Expr> {
```

Comment:
This method can be removed now, it's doing the wrong thing anyway :)

Reply:
Done